scripts/eurostat/life_expectancy_by_age_gender/generate_mcf.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import pandas as pd
+import re
+
+def convert_range(s_input):
+    """convert range values from the format in StatVars to the format as 
```

Comment:
I think Google style prefers docstrings to be in full sentence form: EurostatRegionalStatistics

Reply:
Got it.